libs/client-sdk/README.md

```diff
@@ -57,29 +72,48 @@ build system (CMake for instance).
 Learn more
 -----------
 
-Read our [documentation](https://docs.launchdarkly.com) for in-depth instructions on configuring and using LaunchDarkly. You can also head straight to the [complete reference guide for this SDK](TODO).
+Read our [documentation](https://docs.launchdarkly.com) for in-depth instructions on configuring and using LaunchDarkly.
+You can also head straight to
+the [complete reference guide for this SDK][reference-guide].
```

Comment:
```suggestion
the [complete reference guide for this SDK](https://docs.launchdarkly.com/sdk/client-side/c-c--).
```
I think this is the link you're looking for?

Reply:
Ah, what happened here is I refactored the link out (don't know what that's called in markdown..) so it's at the bottom of the markdown, and now we can refer to it in one place using a unique name `reference-guide`.